src/ulisp.js

```diff
@@ -1661,22 +1661,25 @@ function checkIncomingModuleAPI() {
   ignoredModuleProp('fetchSettings');
 }
 var ASM_CONSTS = {
-  55356: () => { return performance.now(); },  
```

Comment:
No idea why it changed these constants actually. 🤔

Reply:
I think the constants are for internal use. This `src/ulisp.js` is generated by Emscripten when building `src/ulisp.c`, as a JS wrapper/loader for the Wasm binary. Both will get overwritten anyway, the next time `bun run build:web` is run.